daemon/src/wallet.rs

```diff
@@ -33,7 +33,7 @@ use xtra_productivity::xtra_productivity;
 use xtras::SendAsyncSafe;
 use xtras::SendInterval;
 
-const SYNC_INTERVAL: Duration = Duration::from_secs(10);
+const SYNC_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
🤡 We could consider basing this value on the network, but it may not matter too much.

Reply:
What do you mean with this? Sync more often on mainnet?